mopidy_mpd/protocol/__init__.py

```diff
@@ -20,8 +20,8 @@
 #: The MPD protocol uses ``\n`` as line terminator.
 LINE_TERMINATOR = b"\n"
 
-#: The MPD protocol version is 0.19.0.
-VERSION = "0.19.0"
+#: The MPD protocol version is 0.21.0.
+VERSION = "0.21.0"
```

Comment:
we shouldn't advertise 0.21 support until we have implemented (at least partial) support for find/search [filters](https://mpd.readthedocs.io/en/latest/protocol.html#filters).